os/tools/README_download.md

```diff
@@ -0,0 +1,35 @@
+# Zipdownload / Download
+
+This guides how to use *zipdownload.sh* and *download.sh* / *download.bat*.  
+They provide binaries programming without TizenRT source codes.  
+This is useful who does not want to get TizenRT source codes and only wants to program given TizenRT binaries.
+
+## Binary Maker
+
+After building TizenRT, *zipdownload* script zips mandatory files including binaries, download scripts and tools.
```

Comment:
"After building" seems to contradict the claim that the user does not have to build. Please check and ignore if this is not the case.